File: C/Atividade26.c
#include <stdio.h>
#include <conio.h>
#include <stdlib.h>
#include<stdio.h>
#include<time.h> 
#include<string.h>
#include<locale.h>  

int main()
{
	int num,mult,i;
    printf ("Insira um numero positivo:");
    scanf ("%d", &num);

	while (num<=0)
	{
		printf ("Invalido! Insira um numero positivo:");
    	scanf ("%d", &num);
	}	

    printf ("\n\nVoce digitou o numero positivo %d e sua tabuada e:\n\n",num);
	
		for (i=0; i<=10; i++)
	{
		mult=num*i;
		printf ("%d * %d = %d\n",num,i,mult);
	}	
    return 0;
}

File: Python/Atividade26.py
valor1 = float(input("Digite um valor positivo: "))

while (valor1<=0):
    valor1 = float(input("Invalido! Digite um número positivo: "))

print ("A tabuada de", valor1, "é:")

for i in range (1, 11, 1):

    tabuada= valor1*i
    print (valor1,"*", i, "=", tabuada )